src/sage/groups/perm_gps/permgroup.py

```diff
@@ -4106,6 +4106,33 @@ def isomorphism_type_info_simple_group(self):
         else:
             raise TypeError("group must be simple")
 
+    def minimal_normal_subgroups(self):
+        """
+        Return a list containing those nontrivial normal subgroups of the group that are minimal among the nontrivial normal subgroups.
+
+        EXAMPLES::
+
+            sage: G = PermutationGroup([(1,2,3),(4,5)])
+            sage: G.minimal_normal_subgroups()
+            [Subgroup generated by [(4,5)] of (Permutation Group with generators [(4,5), (1,2,3)]),
+             Subgroup generated by [(1,2,3)] of (Permutation Group with generators [(4,5), (1,2,3)])]
+        """
+        return [self.subgroup(gap_group=gap_subgroup) for gap_subgroup in self.gap().MinimalNormalSubgroups()]
+
+    def maximal_normal_subgroups(self):
+        """
+        Return a list containing those proper normal subgroups of the group G that are maximal among the proper normal subgroups.
+        Gives error if G/G' is infinite, yielding infinitely many maximal normal subgroups.
+
+        EXAMPLES::
+
+            sage: G = PermutationGroup([(1,2,3),(4,5)])
+            sage: G.maximal_normal_subgroups()
+            [Subgroup generated by [(1,2,3)] of (Permutation Group with generators [(4,5), (1,2,3)]),
+             Subgroup generated by [(4,5)] of (Permutation Group with generators [(4,5), (1,2,3)])]
+        """
+        return [self.subgroup(gap_group=gap_subgroup) for gap_subgroup in self.gap().MaximalNormalSubgroups()]
```

Comment:
```suggestion
        return [self.subgroup(gap_group=gap_subgroup)
                for gap_subgroup in self.libgap().MaximalNormalSubgroups()]
```

Reply:
Here, libgap() is not working, it is giving an AttributeError. But I found that other functions similar to this one uses _libgap_() so I have implemented that, is it okay ?

Reply:
Yes, that's right. Sorry, I forgot the underscores (`_`).